baremetal/drivers/micron_mt25q/micron_mt25q.c

```diff
@@ -340,7 +408,8 @@ uint8_t Flash_program(uint8_t* buf, uint32_t wr_addr, uint32_t wr_len)
     /*This command works for all modes. No Dummy cycles*/
     /*now program the sector. This will set the desired bits to 0.*/
 
-    command_buf[0] = MICRON_PAGE_PROGRAM;
+    command_buf[0] = 0x02;
```

Comment:
*minor*: The `MICRON_PAGE_PROGRAM` define seems to already be `0x02`. Why was a magic number used versus the definition?

Reply:
Reverted to MICRON_PAGE_PROGRAM for clarity